core/src/main/java/me/lokka30/treasury/plugin/core/command/subcommand/InfoSubcommand.java

```diff
@@ -70,7 +70,7 @@ public void execute(@NotNull CommandSource sender, @NotNull String label, @NotNu
                     placeholder("api-version", provider.getSupportedAPIVersion()),
                     placeholder("supports-bank-accounts", Utils.getYesNoStateMessage(provider.hasBankAccountSupport())),
                     placeholder("supports-transaction-events", Utils.getYesNoStateMessage(provider.hasTransactionEventSupport())),
-                    placeholder("primary-currency", provider.getPrimaryCurrency().getCurrencyName()))
+                    placeholder("primary-currency", provider.getPrimaryCurrency().getPrimaryCurrencyName()))
```

Comment:
maybe add as placeholder "currency_<uuid>_name_<num>" to use a spacific name of a curency?

Reply:
primary name should be shown as the other names may be only for parse ease
e.g. `dollar, dolla, dola, buck, bucks`

Reply:
oh, I see